fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -787,7 +934,91 @@ void applyMetadataMutations(SpanID const& spanContext,
 			toCommit->writeTypedMessage(mutationEnd);
 		}
 	}
-}
+
+public:
```

Comment:
Nit: maybe move constructor to the begining.

Reply:
Done